priv/templates/phx.gen.auth/live_auth.ex

```diff
@@ -0,0 +1,61 @@
+defmodule <%= inspect context.web_module %>.<%= if schema.web_path, do: "#{Phoenix.Naming.humanize(schema.web_path)}.", else: "" %><%= Phoenix.Naming.humanize(schema.singular) %>LiveAuth do
+  import Phoenix.LiveView, only: [assign_new: 3, push_redirect: 2]
+  alias <%= inspect context.web_module %>.Router.Helpers, as: Routes
+
+  @doc """
+  Attaches current_<%= schema.singular %> to `socket` assigns based on <%= schema.singular %>_token or nil if it doesn't.
+
+  ## Examples
+
+      # In the LiveView file
+      defmodule DemoWeb.PageLive do
+        use Phoenix.LiveView
+
+        on_mount {<%= inspect context.web_module %>.<%= if schema.web_path, do: "#{Phoenix.Naming.humanize(schema.web_path)}.", else: "" %><%= Phoenix.Naming.humanize(schema.singular) %>LiveAuth, :mount_current_<%= schema.singular %>}
+      end
+
+  """
+  def mount_current_<%= schema.singular %>(socket, %{"<%=schema.singular %>_token" => <%= schema.singular %>_token}) do
+    socket = assign_new(socket, :current_<%= schema.singular %>, fn ->
```

Comment:
@mad42 I pulled your branch down locally and generated a new `user_live_auth.ex`, similar to the CI checks that are failing for this PR. Running mix format changes this line (and line 46) by moving the `assign_new/3` call onto a new line, like this:
```elixir
    socket =
      assign_new(socket, :current_user, fn ->
        DevApp.Accounts.get_user_by_session_token(user_token)
      end)
```
If you change this template to match that, I think the tests will start passing.

Reply:
Thanks for pointing that 🙏